[2023-08-15T07:16:41.330+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spark_airflow_dag.spark_submit_task manual__2023-08-15T07:16:40.329334+00:00 [queued]>
[2023-08-15T07:16:41.335+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spark_airflow_dag.spark_submit_task manual__2023-08-15T07:16:40.329334+00:00 [queued]>
[2023-08-15T07:16:41.335+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-15T07:16:41.341+0000] {taskinstance.py:1327} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2023-08-15 07:16:40.329334+00:00
[2023-08-15T07:16:41.343+0000] {standard_task_runner.py:57} INFO - Started process 321 to run task
[2023-08-15T07:16:41.346+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'spark_***_dag', 'spark_submit_task', 'manual__2023-08-15T07:16:40.329334+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/spark_jobs_dag.py', '--cfg-path', '/tmp/tmp26h46vhl']
[2023-08-15T07:16:41.347+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask spark_submit_task
[2023-08-15T07:16:41.381+0000] {task_command.py:410} INFO - Running <TaskInstance: spark_airflow_dag.spark_submit_task manual__2023-08-15T07:16:40.329334+00:00 [running]> on host 34dbf8232cde
[2023-08-15T07:16:41.427+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='spark_***_dag' AIRFLOW_CTX_TASK_ID='spark_submit_task' AIRFLOW_CTX_EXECUTION_DATE='2023-08-15T07:16:40.329334+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-15T07:16:40.329334+00:00'
[2023-08-15T07:16:41.435+0000] {base.py:73} INFO - Using connection ID 'spark_local' for task execution.
[2023-08-15T07:16:41.437+0000] {spark_submit.py:340} INFO - Spark-Submit cmd: spark-submit --master localhost:8090 --name arrow-spark spark_processing.py
[2023-08-15T07:16:41.490+0000] {spark_submit.py:491} INFO - /home/***/.local/lib/python3.10/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2023-08-15T07:16:42.379+0000] {spark_submit.py:491} INFO - python3: can't open file '/opt/***/spark_processing.py': [Errno 2] No such file or directory
[2023-08-15T07:16:42.386+0000] {spark_submit.py:491} INFO - 23/08/15 07:16:42 INFO ShutdownHookManager: Shutdown hook called
[2023-08-15T07:16:42.387+0000] {spark_submit.py:491} INFO - 23/08/15 07:16:42 INFO ShutdownHookManager: Deleting directory /tmp/spark-217aa5c6-47b3-4e7a-8076-902828a217be
[2023-08-15T07:16:42.431+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 156, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 422, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master localhost:8090 --name arrow-spark spark_processing.py. Error code is: 2.
[2023-08-15T07:16:42.433+0000] {taskinstance.py:1345} INFO - Marking task as FAILED. dag_id=spark_***_dag, task_id=spark_submit_task, execution_date=20230815T071640, start_date=20230815T071641, end_date=20230815T071642
[2023-08-15T07:16:42.440+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 4 for task spark_submit_task (Cannot execute: spark-submit --master localhost:8090 --name arrow-spark spark_processing.py. Error code is: 2.; 321)
[2023-08-15T07:16:42.464+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-08-15T07:16:42.475+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
